src/callback_handler/crypto.rs

```diff
@@ -11,10 +11,17 @@ struct CertificatePool {
     intermediates: Vec<picky::x509::Cert>,
 }
 
+#[derive(Debug)]
+/// Used as return of verify_certificate()
+pub enum BoolWithReason {
+    True,
+    False(String),
+}
```

Comment:
I think you can reuse the same struct that is defined inside of the Rust SDK

Reply:
Better, yes. Done in https://github.com/kubewarden/policy-evaluator/pull/214/commits/d21868bc1b20db0dc2f763c24f5e39e549aa6b01.